### ISSUE TITLE
Can't visit webui through tcp port after installing docker version latest-v4

### ISSUE THREAD

**Issue by kirincc** [romancin/tinymediamanager-docker#62]
Env: qnap 453Bmini
Using docker-compose
```
version: "2.1"
services:
  tinymediamanager:
    image: romancin/tinymediamanager:latest-v4
    container_name: tinymediamanager
    environment:
      - GROUP_ID=0
      - USER_ID=0
      - TZ=Asia/Shanghai
      - ENABLE_CJK_FONT=1 #support chinese gui
    volumes:
      - /share/CACHEDEV1_DATA/Container/tinymediamanager/config:/config
      - /share/CACHEDEV2_DATA/tmm:/media
    ports:
      - 5800:5800
    extra_hosts:
      - "api.themoviedb.org:52.85.61.40"
      - "image.themoviedb.org:104.16.61.155"
      - "image.tmdb.org:52.85.61.95"
      - "api.tmdb.org:52.85.61.101"
      - "api.thetvdb.com:54.239.200.102"
      - "www.themoviedb.org:52.85.61.101"
    restart: always
```

![image](https://user-images.githubusercontent.com/16648064/118145429-2ebd5a00-b440-11eb-9e80-dab6864d4724.png)
![](https://md-private-w-public-r-1256490109.cos.ap-guangzhou.myqcloud.com/others/f8ba5ff0-9bbb-45b3-a8c1-55fcf63928ef.png)

Logs:
```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.,
[s6-init] ensuring user provided files have correct perms...exited 0.,
[fix-attrs.d] applying ownership & permissions fixes...,
[fix-attrs.d] done.,
[cont-init.d] executing container initialization scripts...,
[cont-init.d] 00-app-niceness.sh: executing... ,
[cont-init.d] 00-app-niceness.sh: exited 0.,
[cont-init.d] 00-app-script.sh: executing... ,
[cont-init.d] 00-app-script.sh: exited 0.,
[cont-init.d] 00-app-user-map.sh: executing... ,
[cont-init.d] 00-app-user-map.sh: exited 0.,
[cont-init.d] 00-clean-logmonitor-states.sh: executing... ,
[cont-init.d] 00-clean-logmonitor-states.sh: exited 0.,
[cont-init.d] 00-clean-tmp-dir.sh: executing... ,
[cont-init.d] 00-clean-tmp-dir.sh: exited 0.,
[cont-init.d] 00-set-app-deps.sh: executing... ,
[cont-init.d] 00-set-app-deps.sh: exited 0.,
[cont-init.d] 00-set-home.sh: executing... ,
[cont-init.d] 00-set-home.sh: exited 0.,
[cont-init.d] 00-take-config-ownership.sh: executing... ,
[cont-init.d] 00-take-config-ownership.sh: exited 0.,
[cont-init.d] 00-xdg-runtime-dir.sh: executing... ,
[cont-init.d] 00-xdg-runtime-dir.sh: exited 0.,
[cont-init.d] 10-certs.sh: executing... ,
[cont-init.d] 10-certs.sh: exited 0.,
[cont-init.d] 10-cjk-font.sh: executing... ,
[cont-init.d] 10-cjk-font.sh: installing CJK font...,
[cont-init.d] 10-cjk-font.sh: fetch http://dl-cdn.alpinelinux.org/alpine/edge/testing/x86_64/APKINDEX.tar.gz,

13/05/2021 23:18:10 passing arg to libvncserver: -rfbportv6

13/05/2021 23:18:10 passing arg to libvncserver: -1

13/05/2021 23:18:10 passing arg to libvncserver: -httpportv6

13/05/2021 23:18:10 passing arg to libvncserver: -1

13/05/2021 23:18:10 passing arg to libvncserver: -desktop

13/05/2021 23:18:10 passing arg to libvncserver: TinyMediaManager

13/05/2021 23:18:10 x11vnc version: 0.9.14 lastmod: 2015-11-14  pid: 767

The XKEYBOARD keymap compiler (xkbcomp) reports:

> Internal error:   Could not resolve keysym XF86FullScreen

Errors from xkbcomp are not fatal to the X server

13/05/2021 23:18:10 Using X display :0

13/05/2021 23:18:10 rootwin: 0x43 reswin: 0x200001 dpy: 0xa6ccfb00

13/05/2021 23:18:10 

13/05/2021 23:18:10 ------------------ USEFUL INFORMATION ------------------

13/05/2021 23:18:10 X DAMAGE available on display, using it for polling hints.

13/05/2021 23:18:10   To disable this behavior use: '-noxdamage'

13/05/2021 23:18:10 

13/05/2021 23:18:10   Most compositing window managers like 'compiz' or 'beryl'

13/05/2021 23:18:10   cause X DAMAGE to fail, and so you may not see any screen

13/05/2021 23:18:10   updates via VNC.  Either disable 'compiz' (recommended) or

13/05/2021 23:18:10   supply the x11vnc '-noxdamage' command line option.

13/05/2021 23:18:10 X COMPOSITE available on display, using it for window polling.

13/05/2021 23:18:10   To disable this behavior use: '-noxcomposite'

13/05/2021 23:18:10 

13/05/2021 23:18:10 Wireframing: -wireframe mode is in effect for window moves.

13/05/2021 23:18:10   If this yields undesired behavior (poor response, painting

13/05/2021 23:18:10   errors, etc) it may be disabled:

13/05/2021 23:18:10    - use '-nowf' to disable wireframing completely.

13/05/2021 23:18:10    - use '-nowcr' to disable the Copy Rectangle after the

13/05/2021 23:18:10      moved window is released in the new position.

13/05/2021 23:18:10   Also see the -help entry for tuning parameters.

13/05/2021 23:18:10   You can press 3 Alt_L's (Left "Alt" key) in a row to 

13/05/2021 23:18:10   repaint the screen, also see the -fixscreen option for

13/05/2021 23:18:10   periodic repaints.

13/05/2021 23:18:10 GrabServer control via XTEST.

13/05/2021 23:18:10 

13/05/2021 23:18:10 Scroll Detection: -scrollcopyrect mode is in effect to

13/05/2021 23:18:10   use RECORD extension to try to detect scrolling windows

13/05/2021 23:18:10   (induced by either user keystroke or mouse input).

13/05/2021 23:18:10   If this yields undesired behavior (poor response, painting

13/05/2021 23:18:10   errors, etc) it may be disabled via: '-noscr'

13/05/2021 23:18:10   Also see the -help entry for tuning parameters.

13/05/2021 23:18:10   You can press 3 Alt_L's (Left "Alt" key) in a row to 

13/05/2021 23:18:10   repaint the screen, also see the -fixscreen option for

13/05/2021 23:18:10   periodic repaints.

13/05/2021 23:18:10 

13/05/2021 23:18:10 XKEYBOARD: number of keysyms per keycode 7 is greater

13/05/2021 23:18:10   than 4 and 51 keysyms are mapped above 4.

13/05/2021 23:18:10   Automatically switching to -xkb mode.

13/05/2021 23:18:10   If this makes the key mapping worse you can

13/05/2021 23:18:10   disable it with the "-noxkb" option.

13/05/2021 23:18:10   Also, remember "-remap DEAD" for accenting characters.

13/05/2021 23:18:10 

13/05/2021 23:18:10 X FBPM extension not supported.

Xlib:  extension "DPMS" missing on display ":0".

13/05/2021 23:18:10 X display is not capable of DPMS.

13/05/2021 23:18:10 --------------------------------------------------------

13/05/2021 23:18:10 

13/05/2021 23:18:10 Default visual ID: 0x21

13/05/2021 23:18:10 Read initial data from X display into framebuffer.

13/05/2021 23:18:10 initialize_screen: fb_depth/fb_bpp/fb_Bpl 24/32/5120

13/05/2021 23:18:10 

13/05/2021 23:18:10 X display :0 is 32bpp depth=24 true color

13/05/2021 23:18:10 

13/05/2021 23:18:10 Listening for VNC connections on TCP port 5900

13/05/2021 23:18:10 

13/05/2021 23:18:10 Xinerama is present and active (e.g. multi-head).

13/05/2021 23:18:10 Xinerama: number of sub-screens: 1

13/05/2021 23:18:10 Xinerama: no blackouts needed (only one sub-screen)

13/05/2021 23:18:10 

13/05/2021 23:18:10 fb read rate: 761 MB/sec

13/05/2021 23:18:10 fast read: reset -wait  ms to: 10

13/05/2021 23:18:10 fast read: reset -defer ms to: 10

13/05/2021 23:18:10 The X server says there are 10 mouse buttons.

13/05/2021 23:18:10 screen setup finished.

13/05/2021 23:18:10 


The VNC desktop is:      1bcd20f20856:0
      

0


******************************************************************************

Have you tried the x11vnc '-ncache' VNC client-side pixel caching feature yet?


The scheme stores pixel data offscreen on the VNC viewer side for faster

retrieval.  It should work with any VNC viewer.  Try it by running:


    x11vnc -ncache 10 ...


One can also add -ncache_cr for smooth 'copyrect' window motion.

More info: http://www.karlrunge.com/x11vnc/faq.html#faq-client-caching


s6-svwait: fatal: timed out

[services.d] starting logmonitor...

[services.d] starting openbox...

[logmonitor] no file to monitor: disabling service...

[openbox] starting...

[services.d] starting app...

[app] starting TinyMediaManager...

[tmmsupervisor] TinyMediaManager not started yet.  Proceeding...

[services.d] done.

```
![](https://md-private-w-public-r-1256490109.cos.ap-guangzhou.myqcloud.com/others/20210513232034.png)
![](https://md-private-w-public-r-1256490109.cos.ap-guangzhou.myqcloud.com/others/20210513231716.png)
When I visited http://192.168.3.2:5800/, it displayed a black screen and then failed.
Reinstall version tag: latest (which means latest v3) works fine.

Any ideas? Thanks!

**Comment by ejkeebler** [comment]
I had this issue when I upgraded from v3 to v4 in the actual application, so I removed it and just ran :latest-v4 and it worked without issue.  Moved to a new docker environment and now when I run latest-v4, I have the same black screen issue.  Any luck?

**Comment by rocdeng** [comment]
Me either. V4.1.1 is working, but latest V4 is not. It keeps rebooting again and again.

**Comment by romancin** [comment]
Fixed by https://github.com/romancin/tinymediamanager4-docker/pull/7

Thank you for opening this issue!

**Comment by kirincc** [comment]
> Fixed by [romancin/tinymediamanager4-docker#7](https://github.com/romancin/tinymediamanager4-docker/pull/7)
> 
> Thank you for opening this issue!

version v4.1.5.1 still not working.
logs:
```
1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890
> Internal error:   Could not resolve keysym XF86FullScreen                                                                                                                                           
Errors from xkbcomp are not fatal to the X server                                                                                                                                                     
[services.d] starting x11vnc...                                                                                                                                                                       
[services.d] starting statusmonitor...                                                                                                                                                                
[x11vnc] starting...                                                                                                                                                                                  
06/06/2021 21:08:55 passing arg to libvncserver: -rfbport                                                                                                                                             
06/06/2021 21:08:55 passing arg to libvncserver: 5900                                                                                                                                                 
06/06/2021 21:08:55 passing arg to libvncserver: -rfbportv6                                                                                                                                           
06/06/2021 21:08:55 passing arg to libvncserver: -1                                                                                                                                                   
06/06/2021 21:08:55 passing arg to libvncserver: -httpportv6                                                                                                                                          
06/06/2021 21:08:55 passing arg to libvncserver: -1                                                                                                                                                   
06/06/2021 21:08:55 passing arg to libvncserver: -desktop                                                                                                                                             
06/06/2021 21:08:55 passing arg to libvncserver: TinyMediaManager                                                                                                                                     
06/06/2021 21:08:55 x11vnc version: 0.9.14 lastmod: 2015-11-14  pid: 804                                                                                                                              
06/06/2021 21:08:55 Using X display :0                                                                                                                                                                
06/06/2021 21:08:55 rootwin: 0x43 reswin: 0x400001 dpy: 0x1de09b00                                                                                                                                    
06/06/2021 21:08:55                                                                                                                                                                                   
06/06/2021 21:08:55 ------------------ USEFUL INFORMATION ------------------                                                                                                                          
[services.d] starting app...                                                                                                                                                                          
[statusmonitor] no file to monitor: disabling service...                                                                                                                                              
06/06/2021 21:08:55 X DAMAGE available on display, using it for polling hints.                                                                                                                        
06/06/2021 21:08:55   To disable this behavior use: '-noxdamage'                                                                                                                                      
06/06/2021 21:08:55                                                                                                                                                                                   
06/06/2021 21:08:55   Most compositing window managers like 'compiz' or 'beryl'                                                                                                                       
06/06/2021 21:08:55   cause X DAMAGE to fail, and so you may not see any screen                                                                                                                       
06/06/2021 21:08:55   updates via VNC.  Either disable 'compiz' (recommended) or                                                                                                                      
06/06/2021 21:08:55   supply the x11vnc '-noxdamage' command line option.                                                                                                                             
06/06/2021 21:08:55 X COMPOSITE available on display, using it for window polling.                                                                                                                    
06/06/2021 21:08:55   To disable this behavior use: '-noxcomposite'                                                                                                                                   
06/06/2021 21:08:55                                                                                                                                                                                   
06/06/2021 21:08:55 Wireframing: -wireframe mode is in effect for window moves.                                                                                                                       
06/06/2021 21:08:55   If this yields undesired behavior (poor response, painting                                                                                                                      
06/06/2021 21:08:55   errors, etc) it may be disabled:                                                                                                                                                
06/06/2021 21:08:55    - use '-nowf' to disable wireframing completely.                                                                                                                               
06/06/2021 21:08:55    - use '-nowcr' to disable the Copy Rectangle after the                                                                                                                         
06/06/2021 21:08:55      moved window is released in the new position.                                                                                                                                
```
